hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequest.java

```diff
@@ -81,7 +81,8 @@ public OMKeyCommitRequest(OMRequest omRequest, BucketLayout bucketLayout) {
 
   @Override
   public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
-    CommitKeyRequest commitKeyRequest = getOmRequest().getCommitKeyRequest();
```

Comment:
Changes looks straightforward   preExecute to invoke  -> preExecute of OMClientRequest to populate Userinfo and remove explicitly populating below via builder which was causing the issue.